app.py

```diff
@@ -103,23 +103,22 @@ def main():
             # Handle the case of no configuration file resulting in a 404 response code
 
             # Update default config with those provided
-            try:
-                with open(".pep8speaks.yml", "r") as stream:
+            with open(".pep8speaks.yml", "r") as stream:
+                try:
                     new_config = yaml.load(stream)
-                    config["ignore"] = new_config["ignore"]
-                    for act in ["opened", "updated"]:
-                        for pos in ["header", "footer"]:
-                            config["message"][act][pos] = new_config[
-                                "message"][act][pos].replace("{name}", author)
-                            config["message"][act][pos] = new_config[
-                                "message"][act][pos].replace("{name}", author)
-                            config["message"][act][pos] = new_config[
-                                "message"][act][pos].replace("{name}", author)
-                            config["message"][act][pos] = new_config[
-                                "message"][act][pos].replace("{name}", author)
-                    config["scanner"]["diff_only"] = new_config["scanner"]["diff_only"]
-            except:  # Bad yml file
-                pass
+                    # overriding the default configuration with the one specified
+                    config = new_config
```

Comment:
Overriding the `config` dictionary might cause problems in [this](https://github.com/kshitij10496/pep8speaks/blob/94b43fcfe551a7a9321c906b13b3edd40bf098bb/app.py#L179-L188) block. Suppose the new config only has the `ignore` key-value pairs.